calico-enterprise/getting-started/compatibility.mdx

```diff
@@ -161,7 +161,9 @@ Kubernetes version support aligns with [upstream Kubernetes](#kubernetes-kubeadm
 
 | {{prodname}} version | TKG version | {{prodname}} support                 | Kubernetes versions    |
 | -------------------- | ----------- | ------------------------------------ | ---------------------- |
+| 3.19                 | ???         | {{prodname}} CNI with network policy | ???                    |
 | 3.18                 | 2.4         | {{prodname}} CNI with network policy | 1.27                   |
+| 3.17                 | 1.6         | {{prodname}} CNI with network policy | 1.23                   |
```

Comment:
```suggestion
```
Don't think we supported TKG in 3.17

Reply:
I assumed that it was a docs error. Changed to `| 3.17                 | N/A        | {{prodname}} 3.17 does not support TKG | N/A                   |`.

Idea being that this was people understand it's definitely not supported, rather than being a docs omission.